kitsune/kbadge/migrations/0002_auto_20181023_1319.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('kbadge', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            "UPDATE badger_badge SET image = CONCAT('uploads/', image)"
```

Comment:
@pmac Do you think we need a `WHERE` clause here? Something like `WHERE image NOT LIKE 'uploads/%'`?

Reply:
That does seem like a good idea.